### DISCUSSION TITLE
2. why my pixel 4 cannot use scene-viewer to load models？

### DISCUSSION THREAD

**Discussion by kenchikuliu** [google/model-viewer#1909] [Q&A]
### Description

Please provide a detailed description of the bug, how to reproduce it, and the
expected behavior.  Always include a code snippet, screenshot, any errors
reported in the console, and the model to help us understand and fix the
problem.

#### Live Demo

https://glitch.com/edit/#!/model-viewer

### Version

- model-viewer: vX.X.X

### Browser Affected

- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [x] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by kenchikuliu** [comment]
why my pixel 4 cannot use scene-viewer to load models？

**Comment by elalish** [comment]
As the issue template requests, can you please provide a detailed description of the bug, how to reproduce it, and the
expected behavior? Best of all is to post a link to the site you are having trouble with so I can reproduce the problem.
